akka-docs/src/main/paradox/stream/operators/ActorSink/actorRef.md

```diff
@@ -18,9 +16,11 @@ This operator is included in:
 
 ## Signature
 
-@@signature [ActorSink.scala](/akka-stream-typed/src/main/scala/akka/stream/typed/scaladsl/ActorSink.scala) { #actorRef }
+Scala
+: @@signature [ActorSink.scala](/akka-stream-typed/src/main/scala/akka/stream/typed/scaladsl/ActorSink.scala) { #actorRef }
 
-@@@
+Java
+: @@signature [ActorSink.scala](/akka-stream-typed/src/main/scala/akka/stream/typed/javadsl/ActorSink.scala) { #actorRef }
```

Comment:
Not a java signature but is the java DSL. Keep or remove?

Reply:
I think it's confusing to show the javadsl signature in Scala. Where we don't have the signature in Java I prefer to mark the 'signature' section with `@@@div { .group-scala }`

Reply:
we should include it as java code, otherwise remove and we can add later

Reply:
Show both or just the scala one. 

scaladsl
```scala
def actorRef[T](
      ref: ActorRef[T], 
      onCompleteMessage: T, 
      onFailureMessage: Throwable => T): Sink[T, NotUsed]
```

javadsl
```scala
def actorRef[T](
      ref: ActorRef[T],
      onCompleteMessage: T,
      onFailureMessage: akka.japi.function.Function[Throwable, T]): Sink[T, NotUsed]
```

Reply:
Just the scala one I'd say - no-one would want to use the javadsl from scala.

Reply:
@raboof for background, when you were away Enno suggested an approach that I then used in https://github.com/akka/akka/pull/27832/files#diff-d1f9506825ab5198d009590b32355046R138

Reply:
@patriknw @raboof It's possible all suggestions are now pushed.